Controllers/AccountController.cs

```diff
@@ -24,17 +27,23 @@ public async Task LogOut()
             .WithRedirectUri(Url.Action("Index", "Index")!)
             .Build();
 
-        await HttpContext.SignOutAsync(Auth0Constants.AuthenticationScheme, authenticationProperties);
+        await HttpContext.SignOutAsync(
+            Auth0Constants.AuthenticationScheme,
+            authenticationProperties
+        );
         await HttpContext.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
     }
 
     public async Task SignUp()
     {
         var authenticationProperties = new LoginAuthenticationPropertiesBuilder()
             .WithParameter("screen_hint", "signup")
-            .WithRedirectUri(Url.Action("CreateProfile", "Index")!)
+            .WithRedirectUri(Url.Action("Profile", "Index")!)
```

Comment:
We have logic that routes your to the correct page based on if the family exists or not. If we always redirect to createProfile directly it can lead to weird behavior especially if their family object is in a weird state

Reply:
Ya thats right. Are you a little unsure if we should do it this way?

Reply:
no I'm not worried about it. taking another look at the IndexController though it looks like both '/profile' and '/createprofile/' handle both of those cases where family is null or not. if family is null, both redirect to the create profile page, and if family is not null both redirect to just the profile page. either way it makes no difference I'm pretty sure, unless I'm overlooking something